exec/vector/src/main/java/org/apache/drill/exec/record/metadata/ColumnMetadata.java

```diff
@@ -182,6 +184,10 @@
 
   int expectedElementCount();
 
+  void setFormatValue(String value);
```

Comment:
Perhaps I missed it, but I didn't see unit tests for this new functionality. I heavily rely on unit tests to catch regressions.

Reply:
I have written tests in the context schema parsing and creation.